docs/docs/workflows/create.md

```diff
@@ -8,32 +8,33 @@ This recording presents the essence of this page. It's recommended to read it in
 
 ---
 
-Creating your cluster requires two steps:
+Creating your cluster happens through multiple phases.
+The most significant ones are:
 
 1. Creating the necessary resources in your cloud environment
 2. Bootstrapping the Constellation cluster and setting up a connection
+3. Installing the necessary Kubernetes components
+
+`constellation apply` handles all this in a single command.
+You can use the `--skip-phases` flag to skip specific phases of the process.
+For example, if you created the infrastructure manually, you can skip the cloud resource creation phase.
 
 See the [architecture](../architecture/orchestration.md) section for details on the inner workings of this process.
 
 :::tip
 If you don't have a cloud subscription, you can also set up a [local Constellation cluster using virtualization](../getting-started/first-steps-local.md) for testing.
 :::
 
-## The *create* step
```

Comment:
There are some links to this and the apply section that need to be fixed. Search for `create.md#`